src/main/java/org/commcare/formplayer/beans/AnswerQuestionRequestBean.java

```diff
@@ -23,6 +23,9 @@ public class AnswerQuestionRequestBean extends SessionRequestBean {
     @Nullable
     private Map<String, Object> answersToValidate;
 
+    @Nullable
+    public String metadataId;
```

Comment:
this should not be needed ?

Reply:
no, it's not needed. thanks for pointing that out. I meant to remove that.